CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ in development
 -----
 * Add initial support for Microsoft Teams via BotFramework (new feature)
 * Add unit tests for all adapters (improvement)
+* Allow `st2chatops` to exit from `st2client` failures.
```

Comment:
Changelog record should belong to likely new `0.9.4` version that'll need also a version bump in `package.json` and Github release + npm publish later.

Reply:
Fixed.